netkan/netkan/metadata.py

```diff
@@ -278,3 +287,16 @@ def _number_compare(v1, v2) -> (int, str, str):
 
         def __str__(self):
             return self.string
+
+
+class CkanGroup:
+
+    def __init__(self, ckan_meta_path, identifier):
+        self.path = Path(ckan_meta_path, identifier)
+
+    def highest_version(self):
+        ckans = (Ckan(p) for p in self.path.glob('**/*.ckan'))
+        highest = max(ckans, default=None, key=lambda ck: ck.version)
+        if highest:
+            return highest.version
+        return None
```

Comment:
This is where a doc string might be handy, explaining what a CkanGroup actually is. It is a group of Ckans, but it's tighter in scope than just a group. Naming is hard! Something like

```python
class CkanGroup:

    """A collection of ckans grouped by their identifier"""

   def ....
```

Reply:
We're already testing the version stuff directly, but it'd be good to have some directly for this function. Save us surprises if we decide to refactor down the track.